README.md

```diff
@@ -21,6 +21,21 @@ Basic usage:
           token: ${{ secrets.GITHUB_TOKEN }}
 ```
 
+### Authentication
+When accessing images in a private Chainguard registry, you will need to create an assumable identity with the `viewer` role, and add a step to set up the `chainctl` prior to running digestabot.
+
+Authentication example:
+
+```yaml
+    - uses: chainguard-dev/setup-chainctl@main
```

Comment:
We should avoid pointing to the main, can you update to the lastest release (v0.4.0)?

thanks

Reply:
i will do in a follow up